src/dom/__tests__/autoCleanup.noProcessEnv.test.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react'
+
+import { ReactHooksRenderer } from '../../types/react'
+
+// This verifies that if process.env is unavailable
+// then we still auto-wire up the afterEach for folks
+describe('skip auto cleanup (no process.env) tests', () => {
+  const originalEnv = process.env
+  let cleanupCalled = false
+  let renderHook: ReactHooksRenderer['renderHook']
+
+  beforeAll(() => {
+    process.env = {
+      ...process.env,
+      get RHTL_SKIP_AUTO_CLEANUP(): string | undefined {
+        throw new Error('expected')
+      }
+    }
```

Comment:
I couldn't remove `process.env` from the test environment all together (other things started to crash), but I could make it error when trying to access the variables.